electron/launcher/src/_reducers/authentication.reducer.js

```diff
@@ -0,0 +1,51 @@
+import * as actionTypes from '../_actions/user.actions.types';
```

Comment:
similarly, is there a way to import from project root instead of relative path?

Reply:
React code now uses absolute path with root being `src/` folder.